tox.ini

```diff
@@ -17,14 +18,14 @@ deps =
     django19: {[django]1.9.x}
     django110: {[django]1.10.x}
     django_master: {[django]master}
+    py26,py27,py32: mock
     py32: coverage==3.7.1
     pytest<3.0.0
     pytest-django<3.0.0
     pytest-cov
     django-money-rates
-commands = make test
+commands = py.test --ds=tests.settings --cov=./djmoney tests
```

Comment:
Yes, because here setuptools integration is not used at all + here I added `skipsdist = true`, which also speeds things a bit